### ISSUE TITLE
Statistics could be more visually appealing - graphical representation of the data over time 

### ISSUE THREAD

**Issue by ldarshan1991** [rcaelers/workrave#311]
**Describe the bug**
When we go to `Statistics`, we see lot of user data like how many breaks did user take on a particular date, how many mouse click did he do on a certain date, what is the daily usage of the user on a certain date. If each category of this data is shown on a timeline, then it would be more useful for the user.
So that he can understand how a particular data which he is interested in (say the number of breaks) has changed over the time and he can improvise himself over it.

**To Reproduce**
1. Go to `Menu`
2. Click on 'Statistics'
3. Notice that user has to select each date to check his usage statistics.

**Expected behavior**
A graphical timeline for each of the statistical data (breaks, mouse click, daily usage) over a period of time.
![image](https://user-images.githubusercontent.com/28981482/123810328-ef739b80-d90f-11eb-9454-dac5a6a60e76.png)
Something very similar to what we see in Github about contribution over a period of time.

**Screenshots**

** Linux (please complete the following information in case you encountered the bug on Linux):**
 - Distribution: [e.g. Ubuntu, Fedora, Debian, ...] 
 - Distribution Version: 
 - Desktop Environment: [e.g. Gnome, KDE, XFCE, MATE, Cinnamon, ...]
 - Workrave Version [e.g. 1.10]

** Windows (please complete the following information in case you encountered the bug on Windows):
 - Windows Version: Windows 10
 - Workrave Version 1.10.1

**Additional context**

**Comment by rcaelers** [comment]
Indeed, that would indeed be a nice improvement. Contribution are welcome :-)
